code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -244,12 +246,14 @@
 /mob/living/simple_animal/hostile/guardian/punch/AttackingTarget()
 	..()
 	if(istype(target, /mob/living))
-		src.say("[src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry]\
-		[src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry]")
+		if(src.cry_wait == 0)
```

Comment:
The idea was for having it be every 5th punch, but yeah I guess a time based cooldown would be better.